yt_dlp/extractor/chaturbate.py

```diff
@@ -113,3 +158,10 @@ def _real_extract(self, url):
             'is_live': True,
             'formats': formats,
         }
+
+    def _real_extract(self, url):
+        video_id, tld = self._match_valid_url(url).group('id', 'tld')
+        extraction = self._extract_from_api(video_id, tld)
+        if extraction is None:
+            extraction = self._extract_from_webpage(video_id, tld)
+        return extraction
```

Comment:
I think we could do this
```suggestion
        return self._extract_from_api(video_id, tld) or self._extract_from_webpage(video_id, tld)
```